content/en/blog/2020/wasmhub-istio

```diff
@@ -0,0 +1,52 @@
+---
+title: Extended and Improved WebAssemblyHub to Bring the Power of WebAssembly to Envoy and Istio
+subtitle: A hub to build, publish, share and deploy WebAssembly Envoy extensions.
+publishdate: 2020-03-13
+attribution: "Idit Levine (Solo.io)"
+keywords: [wasm,extensibility,alpha,performance,operator]
+---
```

Comment:
Note that this is missing a `description:` and if you look at the netlify job output you will not see this blog listed on the 2020 page.

Reply:
It's missing in the index because it's not in the right location. This file needs to be named `index.md` and placed in a directory named `wasmhub-istio`.

Reply:
that is, mv `/blog/2020/wasmhub-istio` to `/blog/2020/wasmhub-istio/index.md`

Reply:
description added